components/atom/image/src/index.js

```diff
@@ -80,13 +82,19 @@ const AtomImage = ({
         className={classNamesFigure}
         style={!error && (placeholder || skeleton) ? figureStyles : {}}
       >
-        <img
-          className={CLASS_IMAGE}
-          onLoad={handleLoad}
-          onError={handleError}
-          ref={imageRef}
-          {...imgProps}
-        />
+        <picture>
+          {sources.map((source, idx) => (
+            <source key={idx} {...source} />
+          ))}
+          <img
+            className={CLASS_IMAGE}
+            onLoad={handleLoad}
+            onError={handleError}
```

Comment:
I see the `handleLoad` function is memoized with useCallback, while the `handleError` function  is redefined for every render. In this case the handleLoad benefit is vanished by the declaration of handleError, which will be every time a new prop for the image tag. It's even possible that using useCallback in this case is a de-optimization, since it has compute that extra piece of logic with no real benefit. I may suggest to measure it and remove the useCallback memoization

Reply:
I'll remove it